### PR TITLE
add: xxxx is not a package

### PR DESCRIPTION
**Pull request by a3510377**
add: xxxx is not a package
for #4

### CODE REVIEW

**Comment by a3510377** [comment]
done

File: docs/python/index.md [hunk 1]
```diff
@@ -190,9 +190,30 @@ FileNotFoundError: [Errno 2] No such file or directory: 'setting.json'
 
 ![img](/imgs/cmds_folder.png)
 
+</Block>
+
 
+## xxxx is not a package
+<Block type="danger" title="問題">
+<div>執行 bot 後抱錯，找不到 discord 函數庫</div>
```
Comment (Proladon) [inline docs/python/index.md:198]: "抱"錯
報錯

File: docs/python/index.md [hunk 2]
```diff
@@ -190,9 +190,30 @@ FileNotFoundError: [Errno 2] No such file or directory: 'setting.json'
...
+ModuleNotFoundError: No module named 'discord.ext'; 'discord' is not a package
+```
+
+![img](/imgs/python/xxxx-is-not-a-package/error-src.jpg)
 
 </Block>
 
+<Block type="success" title="解決方法">
+原因：
+檔案/資料夾 命名與函數庫名稱相同
```
Comment (Proladon) [inline docs/python/index.md:213]: 建議加個範例會比較好
有些小白可能連 `函數庫` / `函式庫` 是甚麼都不懂

例如:
```
原因：
檔案/資料夾 命名與函數庫名稱相同
例: 檔案名稱請勿命名為 discord.py
```

**Review by Proladon — CHANGES_REQUESTED** [review]

File: docs/python/index.md (hunk 1 above)
Comment (a3510377) [reply docs/python/index.md:198]: 痾好尷尬ww

Comment (a3510377) [reply docs/python/index.md]: fix donw: https://github.com/SHELTER-ZONE/Discord_Bot_FAQ/pull/9/commits/fcdc6177e343200bffadd1412017234be1023dac

**Comment by a3510377** [comment]
done

File: docs/python/index.md (hunk 2 above)
Comment (a3510377) [reply docs/python/index.md:218]: fix: https://github.com/SHELTER-ZONE/Discord_Bot_FAQ/pull/9/commits/97423b7684886f083677e170b78760f7e2b5a58c

**Review by Proladon — APPROVED** [review]

**Comment by Proladon** [comment]
@a3510377 有衝突，解一下

**Comment by a3510377** [comment]
done

**Pull request merged**